src/rdmnet/broker/broker_api.cpp

```diff
@@ -0,0 +1,67 @@
+/******************************************************************************
+ * Copyright 2019 ETC Inc.
```

Comment:
It's getting to be the end of the year.  We'll want to remember to update the copyright to 2020 when we're touching files.

Reply:
I was just envisioning scripting a mass update rather than touching files individually.

Reply:
Agreed.  This was a convenient spot to make sure you were reminded. :)